move/its/tests/coin.move

```diff
@@ -0,0 +1,48 @@
+#[test_only]
+module its::coin {
+    use sui::url::{Url};
+    use sui::coin::{Self, CoinMetadata, TreasuryCap};
+
+    public struct COIN has drop{}
+    
+
+    public fun create_treasury_and_metadata(symbol: vector<u8>, decimals: u8, ctx: &mut TxContext): (TreasuryCap<COIN>, CoinMetadata<COIN>) {
```

Comment:
Add section comments (similar to other modules to highlight when sections start)

Reply:
done